src/pages/u/[handle].tsx

```diff
@@ -4,14 +4,51 @@ import withAccessType from '@/lib/hoc/withAccessType';
 import { CookieService, PermissionService } from '@/lib/services';
 import type { PublicProfile } from '@/lib/types/apiResponses';
 import { CookieType } from '@/lib/types/enums';
+import defaultProfilePicture from '@/public/assets/profile-pics/adorable0.png';
+import styles from '@/styles/pages/u.module.scss';
+import Image from 'next/image';
 import type { GetServerSideProps } from 'next/types';
 
 interface UserProfilePageProps {
   user: PublicProfile;
 }
 
 const UserProfilePage = ({ user }: UserProfilePageProps) => {
-  return <pre>{JSON.stringify(user, null, 2)}</pre>;
+  // return <pre>{JSON.stringify(user, null, 2)}</pre>;
```

Comment:
you can delete this fully now that we're actually making it

Reply:
fixed in Clean up code 1